Core/UI/steering/steering.c

```diff
@@ -614,3 +614,177 @@ void handle_ptt_message(void) {
     ecu_ack = true;
   }
 }
+
```

Comment:
As the previous comment, try to move these functions (only the definitions) in tab_debug.c. If there are problems write me.